res/drawable/ic_map_white_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```

Comment:
the map-icon is updated in this pr as at in the first iteration, there was a video-icon as well and the map-icon did not match standard-sizes. later, we decided to remove video-icon - but the map-icon is still nicer :)